controls/datagrid/filtering/filter-control-template.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Filter Control Template
+page_title: .NET MAUI DataGrid Documentation - Filter Control Template
+description: "Review the Telerik .NET MAUI DataGrid Filter Control Template documentation article to learn more about applying custom filter to the DataGrid using FilterControlTemplate property."
+position: 2
+slug: datagrid-filter-control-template
+---
+
+# Filter Control Template
+
+The Telerik DataGrid provides a built-in filtering UI which enables users to quickly filter the displayed data. To customize the built-in filtering UI for a concrete column, use the `FilterControlTemplate` property of the typed columns (text, boolean, numeric, date, time and picker columns). 
+
+In addition, as the template column does not provide a default filtering UI, with the `FilterControlTemplate` property you can allow users to filter data in template columns as well.
```

Comment:
Can you add a reference? Don't assume that everyone knows that we have template columns.